### ISSUE TITLE
Error: $config is not defined

### ISSUE THREAD

**Issue by tobkle** [sst/ion#645]
After installation of sst with ```curl -fsSL https://ion.sst.dev/install | bash```
Receiving the following error:

```zsh
╰─ sst init
╰─ npx sst deploy                                                                                                                                                                                 ─╯
Need to install the following packages:
sst@2.43.3
Ok to proceed? (y) y

Error: $config is not defined

Trace: ReferenceError: $config is not defined
    at file:///Users/.../.sst.config.1720338849962.mjs:4:26
```

sst.config.ts 
```typescript
/// <reference path="./.sst/platform/config.d.ts" />

export default $config({
  app(input) {
    return {
      name: "00",
      removal: input?.stage === "production" ? "retain" : "remove",
      home: "aws",
    };
  },
  async run() {},
});
```

(Environment: Apple M1 Max OS Sonoma 14.5)

**Comment by zohaibakber** [comment]
its not **npx sst dev** its just **sst dev** sst is not installed on your machine as a cli tool

**Comment by 0cv** [comment]
I've got this problem too, it seems a conflicting issue between SST V2 and Ion. Especially, `sst` is installed locally when initializing a new `ion` project with `"sst" : "ion"` in the `package.json` and `ion` is in fact referencing the latest SST version, i.e. v2 at the moment. So when you have a package.json script executing a sst command, you're essentially running sst v2 ... The fix I've found is simply to remove the `sst` entry from `package.json`, `sst` is not referenced from `sst.config.ts` anymore.

Because SST is installed globally, specifying e.g. `sst deploy ...` in the package.json scripts will default back to the globally installed SST (i.e. ion).
